WebAPI/Identity/Auth/Response.cs

```diff
@@ -1,9 +1,13 @@
 ﻿
+using System.Net;
+
 namespace WebAPI.Models.Auth
 {
+    public enum Replay { Yes, No }
```

Comment:
What should this enum do?
Why it is called `Replay`?
Isn't usage of boolean for isX flag better? Especially when you only use 2 values to switch between

Reply:
Changed in Seed User branch